index.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title></title>
```

Comment:
The TITLE tag is usually placed at the end of the HEAD tag and is not left empty.